skt/executable.py

```diff
@@ -44,6 +45,58 @@ def full_path(path):
     return os.path.abspath(os.path.expanduser(path))
 
 
+def destroy_state_file(cfg):
+    """
+    Destroy the state file
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        os.unlink(state_file)
+
+
+def read_state_file(cfg):
+    """
+    Read the state file from the disk
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        with open(state_file, 'r') as fileh:
+            current_state = yaml.load(fileh)
+    else:
+        current_state = {}
+
+    return current_state
+
+
+def save_state_file(cfg, state_updates):
+    """
+    Save state to the state file on disk
+
+    Args:
+        cfg:           A dictionary of skt configuration.
+        state_updates: A dictionary of items to update.
+    """
+    current_state = read_state_file(cfg)
+
+    # Merge the current state and the updates.
+    new_state = current_state.copy()
+    new_state.update(state_updates)
+
+    # Save the new state to the state file
+    logging.debug("Saving state: {}".format(state_updates))
+    state_file = cfg.get('state')
+    with open(state_file, 'w') as fileh:
+        fileh.write(
+            yaml.dump(new_state, default_flow_style=False)
+        )
```

Comment:
Can we have this shortened on a single line since the split formatting is not needed? :)

Reply:
Sure.